test/FluentMigrator.Tests/FluentMigrator.Tests.csproj

```diff
@@ -76,6 +78,10 @@
       <Link>%(Filename)%(Extension)</Link>
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
+    <Content Include="../../lib/Snowflake/*.*">
```

Comment:
Reminder for me: Check if it's still required

Reply:
That folder was removed, so no, it is not. Thanks for accepting the PR!